modules/nw-egress-router-cr.adoc

```diff
@@ -0,0 +1,118 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/deploying-egress-router-ovn-redirection.adoc
+
+ifeval::["{context}" == "deploying-egress-router-ovn-redirection"]
+:redirect:
+:router-type: redirect
+endif::[]
+:router-name: egress-router-{router-type}
+
+[id="nw-egress-router-ovn-cr_{context}"]
+= Egress router custom resource
+
+Define the configuration for an egress router pod in an egress router custom resource. The following YAML describes the fields for the configuration of an egress router in {router-type} mode:
+
+// cluster-network-operator/manifests/0000_70_cluster-network-operator_01_egr_crd.yaml
+[source,yaml,subs="attributes+"]
+----
+apiVersion: network.operator.openshift.io/v1
+kind: EgressRouter
+metadata:
+  name: <egress_router_name>
+spec:
+  addresses: [  <.>
+    {
+      ip: "<egress_router>",  <.>
+      gateway: "<egress_gateway>"  <.>
+    }
+  ]
+  networkInterface: {  <.>
+    macvlan: {
+      master: "<master_interface>",  <.>
+      mode: "<macvlan_mode>"  <.>
+    }
+  }
+  mode: Redirect
+  redirect: {
+    redirectRules: [  <.>
+      {
+        destinationIP: "<egress_destination>",
+        port: <egress_router_port>,
+        targetPort: <target_port>,  <.>
+        protocol: <network_protocol>  <.>
+      },
+      ...
+    ]
+    fallbackIP: "<egress_destination>"  <.>
+  }
+----
+<.> The `addresses` field specifies the IP addresses to configure on the secondary network interface.
```

Comment:
I found the field descriptions from the [crd](https://github.com/openshift/cluster-network-operator/blob/master/manifests/0000_70_cluster-network-operator_01_egr_crd.yaml) in the CNO repo.  Please check.